src/Select/index.tsx

```diff
@@ -533,7 +534,7 @@ export const Select: React.FC<Props> = ({
                   React.isValidElement(triggerAs) &&
                     (triggerAs.props as any).className,
                 ),
-                color: colors.white,
+                color: color ?? colors.white,
```

Comment:
And then just read `color`
```suggestion
                color,
```